### ISSUE TITLE
Unable to bind upgrades to armor

### ISSUE THREAD

**Issue by acokeaday** [TeamBR-Modding/NeoTech#362]
What Version of NeoTech are you using:
Minecraft 1.9 
NeoTech-1.9-3.0.6

If possible, did you break and replace the block:
No issue with this
Please Pastebin.com the crash Report and put link here:
No error nothing happens when binding upgrades to armor in thermal binder

**Comment by medsal15** [comment]
Did you try to use molten tin?

**Comment by acokeaday** [comment]
Hi, Yes
I've used molten tin.  It is full. 
 I've made motherboards with processors stuff.  No issue with those.  

Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: medsal15 notifications@github.com Date: 14/07/2016  00:12  (GMT+08:00) To: TeamBR-Modding/NeoTech NeoTech@noreply.github.com Cc: acokeaday se@n.acokeaday.com, Author author@noreply.github.com Subject: Re: [TeamBR-Modding/NeoTech] Unable to bind upgrades to armor (#362) 
Did you try to use molten tin?
